### DISCUSSION TITLE
MuMu模拟器启动后不能正常打开加速器

### DISCUSSION THREAD

**Discussion by citrusreticulata** [BlueArchiveArisHelper/BAAH#117] [Q&A] [answered]
如题，使用的模拟器为MuMu模拟器，使用的加速器为自带的加速器。区服为国际服（繁中）。

加速器对应的应用为：`com.mumu.acc`。这个应用在模拟器启动后有可能不会正常运行，导致BAAH后续运行失败。

尝试的解决方法，一个是在BAAH的开启加速器选项里面填写`com.mumu.acc`，点击设置为无。该方法似乎不能正常启动加速器。该加速在模拟器内部似乎没有应用，而是需要从模拟器外（世界之外的力量是吧）打开，所以比较麻烦。

尝试二，使用adb调试试图读取`com.mumu.acc`对应的启动activity，但是找不到。。。只能找到包名。

总结，有时打开MuMu模拟器后，自带加速器不会正常启动，导致后续的BAAH会报“服务器正在维护”等错误。

这个大家有遇到过吗？有没有什么比较好的解决方法呢

**Comment by sanmusen214** [comment]
建议模拟器内下个第三方加速器或者clash使用

**Comment by citrusreticulata** [comment, marked as answer]
Mumu模拟器更新后彻底将加速列为收费功能！同时体验是一点没优化，开启游戏时还是随缘加速。

已经彻底转向第三方加速！可以通过BAAH自带的功能智能开启clash，具体教程参考另一个讨论：#317 

赞美BAAH！
